includes/class-wc-stripe-intent-controller.php

```diff
@@ -972,7 +1001,7 @@ private function build_base_payment_intent_request_params( $payment_information
 	 * Determines if mandate data is required for deferred intent UPE payment.
 	 *
 	 * A mandate must be provided before a deferred intent UPE payment can be processed.
-	 * This applies to SEPA, Bancontact, iDeal, Sofort, Cash App and Link payment methods.
+	 * This applies to SEPA, Bancontact, iDeal, Sofort, Cash App, Link payment methods and ACSS Debit.
```

Comment:
Can we add the missing payment methods here too?

```suggestion
	 * This applies to SEPA, Bancontact, iDeal, Sofort, Cash App, Link payment methods,
	 * ACH, ACSS Debit and BACS.
```